src/app/components/NavBar/index.tsx

```diff
@@ -33,33 +33,50 @@ export const Dropdown: FunctionComponent<{user: User, currentPath: string}> = ({
     }
   };
 
-  return <OnScroll callback={blockScroll}>
-    <Styled.DropdownOverlay tabIndex='-1' ref={overlay} data-testid='nav-overlay'>
-      <a aria-hidden='true' tabIndex={-1} href='/'>
-        <Styled.OverlayLogo
-          src={openstaxLogo}
-          alt={useIntl().formatMessage({id: 'i18n:nav:logo:alt'})}
-        />
-      </a>
-      <div>
-        <FormattedMessage id='i18n:nav:hello:text' values={{name: user.firstName}}>
-          {(msg) => <Styled.OverlayHeading>{msg}</Styled.OverlayHeading>}
-        </FormattedMessage>
-        <Styled.DropdownList>
-          <li>
-            <FormattedMessage id='i18n:nav:profile:text'>
-              {(msg) => <a href='/accounts/profile' target='_blank'>{msg}</a>}
-            </FormattedMessage>
-          </li>
-          <li>
-            <FormattedMessage id='i18n:nav:logout:text'>
-              {(msg) => <a href={'/accounts/logout?r=' + currentPath}>{msg}</a>}
-            </FormattedMessage>
-          </li>
-        </Styled.DropdownList>
-      </div>
-    </Styled.DropdownOverlay>
-  </OnScroll>;
+  return (
+    <OnScroll callback={blockScroll}>
+      <Styled.DropdownOverlay
+        tabIndex='-1'
+        ref={overlay}
+        data-testid='nav-overlay'
+      >
+        <a aria-hidden='true' tabIndex={-1} href='/'>
+          <Styled.OverlayLogo
+            src={openstaxLogo}
+            alt={useIntl().formatMessage({ id: 'i18n:nav:logo:alt' })}
+          />
+        </a>
+        <div>
+          <FormattedMessage
+            id='i18n:nav:hello:text'
+            values={{ name: user.firstName }}
+          >
+            {msg => <Styled.OverlayHeading>{msg}</Styled.OverlayHeading>}
+          </FormattedMessage>
+          <Styled.DropdownList id='dropdown-menu' role='menu'>
+            <li role='presentation'>
+              <FormattedMessage id='i18n:nav:profile:text'>
+                {msg => (
+                  <a href='/accounts/profile' target='_blank' role='menuitem'>
+                    {msg}
+                  </a>
+                )}
+              </FormattedMessage>
+            </li>
+            <li role='presentation'>
+              <FormattedMessage id='i18n:nav:logout:text'>
+                {msg => (
+                  <a href={'/accounts/logout?r=' + currentPath} role='menuitem'>
+                    {msg}
```

Comment:
also added role attribute here (and the `presentation` role to the enclosing `li`)